content/engineering/our-approach/people/assessment.md

```diff
@@ -10,7 +10,7 @@ layout: layouts/page
 eleventyNavigation: 
   parent: engineering_approach
   key: Assessment guide
-  order: 10
+  order: 12
```

Comment:
three different pages currently have `order: 10`, which isn't helpful for setting page order!